server/cluster/metadata/table_manager.go

```diff
@@ -298,3 +325,25 @@ func (m *TableManagerImpl) getTable(schemaName, tableName string) (storage.Table
 	table, ok := tables.tables[tableName]
 	return table, ok, nil
 }
+
+func (m *TableManagerImpl) getTables(schemaName string, tableNames []string) ([]storage.Table, error) {
+	schema, ok := m.schemas[schemaName]
+	if !ok {
+		return []storage.Table{}, ErrSchemaNotFound.WithCausef("schema name", schemaName)
+	}
+
+	tables, ok := m.schemaTables[schema.ID]
+	if !ok {
+		return []storage.Table{}, nil
+	}
+
+	ret := make([]storage.Table, 0, len(tableNames))
```

Comment:
Such meaningless name `ret` should be avoided.
```suggestion
	tables := make([]storage.Table, 0, len(tableNames))
```